components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -296,19 +308,45 @@ public List<BasicOrganization> getUserAuthorizedOrganizations(Integer limit, Str
                                                                   String applicationAudience)
             throws OrganizationManagementException {
 
-        return getOrganizationList(true, limit, after, before, sortOrder, filter, recursive, applicationAudience);
+        return getBasicOrganizationList(true, limit, after, before, sortOrder, filter, recursive,
+                applicationAudience);
     }
 
-    private List<BasicOrganization> getOrganizationList(boolean authorizedSubOrgsOnly, Integer limit, String after,
+    private List<BasicOrganization> getBasicOrganizationList(boolean authorizedSubOrgsOnly, Integer limit, String after,
```

Comment:
What is the meaning of getting the basic organization list?

Reply:
This is now changed to getOrganizationsBasicInfo.